doc/dependencies.md

```diff
@@ -1,8 +1,8 @@
 # Dependencies
 
 These are the dependencies used by Bitcoin Core.
-You can find installation instructions in the `build-*.md` file for your platform.
-"Runtime" and "Version Used" are both in reference to the release binaries.
+You can find installation instructions in the `build-*.md` file for your platform, or self-compile
```

Comment:
In commit 0a6547d26b570625922478e7e993ad1c97ec3ab4 maybe reference from root (feel free to ignore)

```suggestion
You can find installation instructions in the `/doc/build-*.md` file for your platform, or self-compile
```